### PR TITLE
Create MeetingNotes.txt

### PR DESCRIPTION
**Pull request by finndobbs**
2019-09-11

### CODE REVIEW

File: MeetingNotes.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
+-we will use gdk
+
```
Comment (Vica98) [inline MeetingNotes.txt]: I'm testing the comments

File: MeetingNotes.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
+-we will use gdk
```
Comment (zjp) [inline MeetingNotes.txt]: GTK

File: MeetingNotes.txt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  Ships (ship class)
+    --variables--
+      size
+      orientation
+      location
+    --Methods--
+      
+    how to place ships
+    
+  Player (player class)
```
Comment (AnjaliPare) [inline MeetingNotes.txt]: Testing the comments

File: MeetingNotes.txt (hunk 2 above)
Comment (sindhushakamuri) [inline MeetingNotes.txt]: Test

**Pull request merged**